rig/machine_control/machine_controller.py

```diff
@@ -909,27 +970,25 @@ def flood_fill_aplx(self, *args, **kwargs):
         :py:func:`~rig.place_and_route.util.build_application_map`.
 
         .. warning::
+
             The loading process is likely, but not guaranteed, to succeed.
             This is because the flood-fill packets used during loading are not
-            guaranteed to arrive. The effect of this is one of the following:
-
-            * Some regions may be included/excluded incorrectly.
-            * Some chips will not receive the complete application binary and
-              will silently not execute the binary.
+            guaranteed to arrive. The effect of this is that some chips may not
+            receive the complete application binary and will silently ignore
+            the application loading request.
 
             As a result, the user is responsible for checking that each core
             was successfully loaded with the correct binary. At present, the
             two recommended approaches to this are:
 
             * The user should check that the correct number of application
-              binaries reach their initial barrier (SYNC0), when this facility
-              is used. This is not fool-proof but will flag up all but
-              situations where exactly the right number, but the wrong
-              selection of cores were loaded. (At the time of writing, this
-              situation is not possible but will become a concern in future
-              versions of SC&MP.
+              binaries reach the initial barrier (e.g. ``wait``), when the
+              ``wait`` argument is ``True``. If the number does not match the
+              expected number of cores to be loaded, the next approach must be
+              used:
```

Comment:
If the ``wait`` argument is given then the user should check that the correct number of application binaries reach the initial barrier (e.g., the ``wait`` state). If the number does not match the expected number of loaded cores the next approach must be used: